MANIFEST.in

```diff
@@ -3,14 +3,18 @@ include LICENSE
 include RELEASE.md
 include README.md
 include setup.py
+include pyproject.toml
 
 graft doc
-prune doc/build
-
 graft LICENSES
-
 graft pandas
 
+prune doc/build
+prune asv_bench
```

Comment:
Any reason to include these? We could be even a bit stricter about what we include.

Maybe I should just revert these changes for now.

Reply:
I think we need scripts for doctoring validation. Rest are probably OK to exclude

Reply:
Note these are for the sdist. Do we care about people installing from the sdist, rather than GitHub, being able to run docstring validation?

Reply:
Maybe not. `pd.test()` would no longer have the the `scripts/tests` files available so probably OK so long as we don't set expectations elsewhere that those files should be run as part of a typical purest run